### PR TITLE
Solving I2989

### PR DESCRIPTION
**Pull request by j0rt3g4**
Solving Issue #2989
Removing Lync for SfB.

### CODE REVIEW

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Skype/SfbServer/plan-your-deployment/high-availability-and-disaster-recovery/disaster-recovery.md [hunk 1]
```diff
@@ -51,7 +51,7 @@ For pool failover and pool failback, the engineering target for recovery time ob
...
   
-All RTO and RPO numbers in this document assume that the two data centers are located within the same world region with high-speed, low-latency transport between the two sites. These numbers are measured for a pool with 40,000 concurrently active users and 200,000 users enabled for Lync with respect to a pre-defined user model where there is no backlog in data replication. They are subject to change based on performance testing and validation.
+All RTO and RPO numbers in this document assume that the two data centers are located within the same world region with high-speed, low-latency transport between the two sites. These numbers are measured for a pool with 40,000 concurrently active users and 200,000 users enabled for Skype for business with respect to a pre-defined user model where there is no backlog in data replication. They are subject to change based on performance testing and validation.
```
Comment (JohanFreelancer9) [inline Skype/SfbServer/plan-your-deployment/high-availability-and-disaster-recovery/disaster-recovery.md]:
```suggestion
All RTO and RPO numbers in this document assume that the two data centers are located within the same world region with high-speed, low-latency transport between the two sites. These numbers are measured for a pool with 40,000 concurrently active users and 200,000 users enabled for Skype for Business with respect to a pre-defined user model where there is no backlog in data replication. They are subject to change based on performance testing and validation.
```

**Comment by scanum** [comment]
@LanaChin PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by scanum** [comment]
@LanaChin  Hi! Following up on this issue, please let me know if there is anything I can do to help!.

**Pull request merged**